api/src/analysis.rs

```diff
@@ -227,17 +228,52 @@ async fn analyze_package_inner(
       .find(|file| file.0.case_insensitive().is_readme());
 
     (
-      generate_score(main_entrypoint, &doc_nodes, &readme, all_fast_check),
+      generate_score(
+        main_entrypoint.clone(),
+        &doc_nodes,
+        &readme,
+        all_fast_check,
+      ),
       readme.map(|readme| readme.0.clone()),
     )
   };
 
   let doc_nodes_json = serde_json::to_vec(&doc_nodes).unwrap().into();
 
+  let info = crate::docs::get_docs_info(&exports, None);
+
+  let ctx = crate::docs::get_generate_ctx(
+    doc_nodes,
+    main_entrypoint,
+    info.rewrite_map,
+    scope,
+    name,
+    version,
+    true,
+    None,
+    false,
+    crate::db::RuntimeCompat {
+      browser: None,
+      deno: None,
+      node: None,
+      workerd: None,
+      bun: None,
+    },
+    registry_url.to_string(),
+  );
+  let search_index = deno_doc::html::generate_search_index(&ctx);
+  let doc_search_json = if let serde_json::Value::Object(mut obj) = search_index
```

Comment:
```suggestion
  let doc_search_json = if let serde_json::Value::Object(mut obj) = search_index
```

You can replace this with `search_index.as_object().unwrap()`.

Reply:
no, it returns a reference